mycroft/api/__init__.py

```diff
@@ -362,49 +358,25 @@ def get_oauth_token(self, dev_cred):
             "path": "/" + self.identity.uuid + "/token/" + str(dev_cred)
         })
 
-    def get_skill_settings(self):
-        """ Fetch all skill settings. """
-        with DeviceApi._skill_settings_lock:
-            if (DeviceApi._skill_settings is None or
-                    time.monotonic() > DeviceApi._skill_settings[0] + 30):
-                DeviceApi._skill_settings = (
-                    time.monotonic(),
-                    self.request({
-                        "method": "GET",
-                        "path": "/" + self.identity.uuid + "/skill"
-                        })
-                )
-            return DeviceApi._skill_settings[1]
+    def get_remote_skill_settings(self):
```

Comment:
I think the "remote" part here is redundant since it's in the DeviceApi it's implied that it's a remote thing.